app/ide-desktop/lib/dashboard/src/providers/AuthProvider.tsx

```diff
@@ -580,28 +437,31 @@ export default function AuthProvider(props: AuthProviderProps) {
       gtagEvent('cloud_sign_out')
       cognito.saveAccessToken(null)
       localStorage.clearUserSpecificEntries()
-      setInitialized(false)
       sentry.setUser(null)
-      setUserSession(null)
+
+      // setUserSession(null)
       // If the User Menu is still visible, it breaks when `userSession` is set to `null`.
       unsetModal()
       // This should not omit success and error toasts as it is not possible
       // to render this optimistically.
-      await toast.toast.promise(cognito.signOut(), {
-        success: getText('signOutSuccess'),
-        error: getText('signOutError'),
-        pending: getText('loggingOut'),
-      })
+      await toast.toast.promise(
```

Comment:
i guess this is what we want to replace with a fullscreen uncloseable dialog

Reply:
yeah